src/pages/landing/[landingPage]/index.tsx

```diff
@@ -0,0 +1,177 @@
+                                key={index}
+                                rel="noreferrer noopener"
+                                href={`${doc.file.url}`}>
+                                {doc.title}
+                              </Link>
+                            )
+                          })}
+                        </div>
+                      ),
+                    },
+                  ]}
+                />
+              )
+            })}
+          </div>
+        )}
+
+        {collections.length >= 1 && <h2 id="collections">Collections</h2>}
+
+        {collections.length >= 1 && (
+          <div className={styles.collectionContainer}>
+            <GridContainer className="grid-container--fill">
+              <Grid row gap>
+                {collections.map(
+                  (collection: CollectionRecord, index: number) => {
+                    return (
+                      <Grid
+                        className={styles.collectionWrapper}
+                        key={`${index}__${collection}`}
+                        mobile={{ col: 12 }}
+                        tablet={{ col: 12 }}
+                        desktop={{ col: 6 }}
+                        desktopLg={{ col: 4 }}>
+                        <Collection title={collection.title}>
+                          {collection.bookmarks.map(
+                            (bookmark: CMSBookmark, index: number) => {
+                              return (
+                                <Bookmark
+                                  key={`${index}__${bookmark.label}`}
+                                  bookmarkDescription={bookmark.description}
+                                  href={bookmark.url}>
+                                  {bookmark.label}
+                                </Bookmark>
+                              )
+                            }
+                          )}
+                        </Collection>
+                      </Grid>
+                    )
+                  }
+                )}
+              </Grid>
+            </GridContainer>
+          </div>
+        )}
+
+        {articles.length >= 1 && <h2 id="articles">Articles</h2>}
+        {articles.length >= 1 && (
+          <ArticleList articles={articles} landingPage={true} />
+        )}
+      </>
+    )
+  }
+
+  return (
+    <Grid className={styles.inPageNav}>
+      <InPageNavigation scrollOffset="140px" content={pageContent()} />
+    </Grid>
+  )
+}
+
+export default LandingPage
+
+LandingPage.getLayout = (page: JSX.Element) =>
+  withLandingPageLayout(page, page.props.landingPage.pageTitle)
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const { landingPage: slug } = context.query
+
+  const session = await getSession(context.req, context.res)
+  const user = session?.passport?.user
+
+  if (!user) {
+    return {
+      redirect: {
+        destination: '/login',
+        permanent: false,
+      },
+    }
+  }
```

Comment:
Question: Did we need to do a redirect here? Asking since I don't know that we do it elsewhere.

Reply:
Good catch, I don't think we do. I'm currently working on adding tests on another branch and I can remove this there.